linchpin/shell/__init__.py

```diff
@@ -336,8 +340,23 @@ def fetch(ctx, fetch_type, remote, root):
               metavar='TX_ID', required=False,
               help="Display a specific transaction by ID (tx_id)."
                    " Only works with '--view=tx'")
+@click.option('--output-format', type=str, default="cfg",
+              metavar='output_format', required=False,
+              help="Inventory output format")
+@click.option('--output-file', type=str,
```

Comment:
I've said this a couple times previously, but we should move this up into the generic options. It should include `--output-pinfile` with this option `--output-file`.